### PR TITLE
Fetch only active user

### PR DESCRIPTION
**Pull request by mufti1**
good afternoon mas @giosakti 

regarding this #162 , I've solved this problem, you can check my PR

happy weekend

### CODE REVIEW

**Comment by giosakti** [comment]
hi @mufti1 thanks for the PR, but please make it more explicit using `'true'` string instead of `1`
Or can the API handle boolean type instead?

**Comment by mufti1** [comment]
@giosakti i've update my PR, I use this approach to allow user using true false or 1 and 0

File: api_blueprint/user.apib [hunk 1]
```diff
@@ -127,6 +127,7 @@ Get user data with email, uid or username
             + Default: bar@test.com
         + uid (string, optional)
         + username (string, optional)
+        + active (int, optional)
```
Comment (giosakti) [inline api_blueprint/user.apib]: for api blueprint, is there a way to specify if a field can be supplied by several different type? (in our case it accepts int, boolean and string right?)

Comment (mufti1) [reply api_blueprint/user.apib]: @giosakti I've updated my PR

**Pull request merged**